bionic/util.py

```diff
@@ -312,6 +313,47 @@ def exception(self, msg, *args, **kwargs):
             logging.ERROR, msg, *args, exc_info=True, **kwargs)
 
 
+class FileCopier(object):
+    """
+    A wrapper for a Path object
+
+
+    Parameters
+    ----------
+    name: String
```

Comment:
We don't need this anymore, do we?

Reply:
Oh why not?  I think it's helpful to have a name associated with a `FileCopier` so the user can easily see which entity it refers to?

Reply:
Isn't that fairly apparent from the file path itself now?

But also, doesn't this feel more like a generic utility object than something that knows about entities, etc.?

Reply:
Bump.

Reply:
Good point! removed the `name` attribute and accompanying reference in docstring to "entity"